### PR TITLE
Less4

### PR DESCRIPTION
**Pull request by frayeralex**
setup gulp
setup font
change primary button style

### CODE REVIEW

**Comment by frayeralex** [comment]
@maksymz revert *.sample files

File: app/design/frontend/Boyko/dada/web/css/source/_typography.less [hunk 1]
```diff
@@ -0,0 +1,8 @@
+& when (@media-common = true) {
```
Comment (maksymz) [inline app/design/frontend/Boyko/dada/web/css/source/_typography.less]: This file overrides https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/blank/web/css/source/_typography.less
You should either preserve original content or use some other file, for example `_extend.less` that is extension point for themes: https://github.com/magento/magento2/blob/2.2.6/lib/web/css/source/_extend.less

Comment (frayeralex) [reply app/design/frontend/Boyko/dada/web/css/source/_typography.less]: done

File: app/design/frontend/Boyko/dada/web/css/source/_extend.less [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// variables
+@dada-font-family: 'Gotham';
+
+@dada-button-primary__background: #9400d3;
+@dada-button-primary__border: 1px solid #000080;
+@dada-button-primary__color: #fff;
+
+// include fonts
+& when (@media-common = true) {
```
Comment (maksymz) [inline app/design/frontend/Boyko/dada/web/css/source/_extend.less]: Variables definition should not be mixed with styles. There are 3 options here:
1) define all variables first, than styles
2) define all variables, import additional .scss files with styles that use these variables (this is better than # 1)
3) move styles for dropdown to:
   a) `_module.less` in your module if you do not need that new variables from the theme
   b) `_extend.less` file for your module in the theme if you need new less variables defined in the theme.
Variant # 3 is the best one.

Comment (frayeralex) [reply app/design/frontend/Boyko/dada/web/css/source/_extend.less]: thank's, i will fix it ASAP

**Pull request merged**